File: public/examples/integrations/mcp-servers/pylon/search_issues_example_call_tool.js
import { Arcade } from "@arcadeai/arcadejs";

const client = new Arcade();
const USER_ID = "{arcade_user_id}";
const TOOL_NAME = "Pylon.SearchIssues";

const toolInput = {
  query: "login timeout",
  auto_accept_matches: false,
};

const response = await client.tools.execute({
  tool_name: TOOL_NAME,
  input: toolInput,
  user_id: USER_ID,
  secrets: { PYLON_API_TOKEN: "<your_pylon_admin_token>" },
});

console.log(JSON.stringify(response.output.value, null, 2));

File: public/examples/integrations/mcp-servers/pylon/search_issues_example_call_tool.py
import json
from arcadepy import Arcade

client = Arcade()

USER_ID = "{arcade_user_id}"
TOOL_NAME = "Pylon.SearchIssues"

tool_input = {
    "query": "login timeout",
    "auto_accept_matches": False,
}

response = client.tools.execute(
    tool_name=TOOL_NAME,
    input=tool_input,
    user_id=USER_ID,
    secrets={"PYLON_API_TOKEN": "<your_pylon_admin_token>"},
)

print(json.dumps(response.output.value, indent=2))
